### PR TITLE
Website update

### PR DESCRIPTION
**Pull request by kathsherratt**
Major content changes
- Added detail on Round 0 results and removed submission text
- Created a new page for community, which mentions the meeting schedule and links to meeting presentations/agendas
- Edited contact page to add info on the mailing list

Cosmetic changes - very happy to take feedback + improvements welcome!
- Added tabs throughout scenario and background pages, to make long blocks of text easier to read. I love tabs so might be overkill
- Added Community as a header section with a user favicon

FYI at least for me, the scenario favicon (looks like it's supposed to be `fa-telescope`)  doesn't display on my browser (chrome).

### CODE REVIEW

File: contact.Rmd [hunk 1]
```diff
@@ -6,4 +6,10 @@ output:
     toc_float: false
 ---
 
-To get in touch, send us an email to [mail@covid19scenariohub.eu](mailto:mail@covid19scenariohub.eu).
+For general information about the European Scenario Hub, our [Github repository](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/issues/new) is typically the most up to date source. For updates on new scenario rounds, upcoming meetings, and results, all are welcome to join the European Scenario Hub [mailing list](https://groups.google.com/u/0/g/euro-covid19-scenario-hub). Please include your name and a brief sentence about your interest in the Hub (so that we can exclude bots).
```
Comment (Bisaloo) [inline contact.Rmd:9]: Should we mention slack as well?

**Comment by Bisaloo** [comment]
And yes, you are correct, the telescope icon doesn't appear because it was introduced in a version more recent than what rmarkdown bundles by default.
I was planning to fix it at some point by force loading a more recent version but please feel free to use another icon if you have something in mind that could fit well.

File: contact.Rmd (hunk 1 above)
Comment (kathsherratt) [reply contact.Rmd:9]: Yes - would you suggest adding the invite link directly?

Comment (Bisaloo) [reply contact.Rmd:9]: Probably not but we should mention it exists. I think there is also a link where they can request access, like a google group. But maybe I've dreamt it :shrug:

Comment (kathsherratt) [reply contact.Rmd]: It looks like Slack links expire after 30 days - at least from what I can see in Slack. So i've just added a line mentioning it. Thanks for flagging!

**Comment by kathsherratt** [comment]
@Bisaloo please could you merge this if you're happy with the changes? Thanks again!

**Comment by sbfnk** [comment]
Whilst we are on the web sites, both the forecast hub and scenario hub pages are a bit impersonal. Shall we add our names somewhere, e.g. to the contact page, perhaps with links to the LSHTM profiles (which have our email addresses)?

**Comment by kathsherratt** [comment]
Turns out neither Hugo nor I have LSHTM profile pages, so I've linked to the epiforecasts team profiles  (and twitter to be a bit less formal).

File: contact.Rmd [hunk 2]
```diff
@@ -17,3 +17,5 @@ We welcome enquiries about the European Scenario Hub and we are keen to hear fro
 - Open a [Github Issue](https://github.com/covid19-forecast-hub-europe/covid19-scenario-hub-europe/issues/new)
 
 We also have a [Slack](https://slack.com/) workspace for all those interested in the Hub - please get in touch for the invite link.
+
+Both the European Scenario and Forecast Hubs are run in collaboration between the [Epiforecasts team](http://epiforecasts.io/), part of the [Centre for Mathematical Modelling of Infectious Diseases](http://cmmid.github.io/topics/covid19) at the [London School of Hygiene & Tropical Medicine](http://www.lshtm.ac.uk), and the [European Centre for Disease Control and Prevention (ECDC)](http://ecdc.europa.eu). The LSHTM team working on Hub research and development includes [Katharine Sherratt](https://epiforecasts.io/people.html#katharine-sherratt) ([@kathsherratt](https://twitter.com/kathsherrratt)), [Hugo Gruson](https://epiforecasts.io/people.html#hugo-gruson) ([@grusonh](https://twitter.com/grusonh), and [Sebastian Funk](https://epiforecasts.io/people.html#sebastian-funk) ([@sbfnk](https://twitter.com/sbfnk)).
```
Comment (Bisaloo) [inline contact.Rmd]:
```suggestion
Both the European Scenario and Forecast Hubs are run in collaboration between the [Epiforecasts team](http://epiforecasts.io/), part of the [Centre for Mathematical Modelling of Infectious Diseases](http://cmmid.github.io/topics/covid19) at the [London School of Hygiene & Tropical Medicine](http://www.lshtm.ac.uk), and the [European Centre for Disease Control and Prevention (ECDC)](http://ecdc.europa.eu). The LSHTM team working on Hub research and development includes [Katharine Sherratt](https://epiforecasts.io/people.html#katharine-sherratt) ([@kathsherratt](https://twitter.com/kathsherrratt)), [Hugo Gruson](https://epiforecasts.io/people.html#hugo-gruson) ([@grusonh](https://twitter.com/grusonh)), and [Sebastian Funk](https://epiforecasts.io/people.html#sebastian-funk) ([@sbfnk](https://twitter.com/sbfnk)).
```

**Pull request merged**